pom.xml

```diff
@@ -186,6 +186,7 @@ xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xs
     <okhttp.version>2.7.5</okhttp.version>
     <mockito1-powermock.version>1.10.19</mockito1-powermock.version>
     <mockito2.version>2.28.2</mockito2.version>
+    <mockito-junit-jupiter.version>2.28.2</mockito-junit-jupiter.version>
```

Comment:
We can just use mockito2.version for mockito-junit-jupiter depedency. AFAIR, mockito-junit-jupiter is part of mockito release cycle, so these versions (mockito2 and mockito-junit-jupiter) should always be the same.

Reply:
Thanks for the suggestion.